sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -91,6 +91,10 @@ public ContactDirectorySteps(
           openContactFromResultsByUUID(apiState.getCreatedContact().getUuid());
         });
 
+    When(
+        "I open the first contact",
```

Comment:
let's be more specific with this action since a contact may be selected from multiple places